### PR TITLE
Add conclusion class

### PR DESCRIPTION
**Pull request by Zclarkwilliams**

### CODE REVIEW

File: src/gameoflife.py [hunk 1]
```diff
@@ -101,8 +111,8 @@ def main():
     args = parser.parse_args()
     
     # set iteration count
-    frames = 100
-    if args.frames and int(args.frames) > 100:
+    frames = 1000
```
Comment (scott-pickthorn) [inline src/gameoflife.py:114]: why start this at 1000 thats too many iterations

File: src/gameoflife.py [hunk 2]
```diff
@@ -34,13 +42,11 @@ def getGrid(grid, N):
...
                 if grid[i][j].life == ALIVE:
                     newGrid[i,j,k] = grid[i][j].state[k]
-    
     if k == 0:
         livecount += 1
-    print ((livecount/N)*100)
     return newGrid
 
-def update(frameNum, img, N, grid):
+def update(frameNum, img, N, grid, totalFrames):
```
Comment (scott-pickthorn) [inline src/gameoflife.py:49]: totalFramess is not used

**Review by scott-pickthorn — APPROVED** [review]

File: src/gameoflife.py (hunk 1 above)
Comment (Zclarkwilliams) [reply src/gameoflife.py:114]: just resetting back to what you submitted it originally to

File: src/gameoflife.py (hunk 2 above)
Comment (Zclarkwilliams) [reply src/gameoflife.py:49]: removed

**Pull request merged**